pkg/plugin/probes_handler.go

```diff
@@ -0,0 +1,17 @@
+package plugin
+
+import (
+	"log"
+	"net/http"
+)
+
+// NewProbesHandler registers liveliness and readinesss probes for /version endpoint
+func NewProbesHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		_, err := w.Write([]byte("200"))
+		if err != nil {
```

Comment:
Can we return plugin version as part of the response and have simple http test for that too?

Reply:
@hemanik this could help https://blog.questionable.services/article/testing-http-handlers-go/